cmd/fs-v1.go

```diff
@@ -854,6 +854,15 @@ func (fs *FSObjects) getObjectInfoNoFSLock(ctx context.Context, bucket, object s
 		return fsMeta.ToObjectInfo(bucket, object, fi), nil
 	}
 
+	if !globalCLIContext.StrictS3Compat {
```

Comment:
Does this mean we are not going to return ETag in listing ?

Reply:
That is exactly what this PR means `--no-compat` @vadmeste